components/simulation-form/NewSimulationForm.tsx

```diff
@@ -330,6 +334,29 @@ export default function NewSimulationForm(props: Props) {
               each existing infection. Leave this field blank to let each model
               decide its own R<sub>0</sub>.
             </p>
+
+            <h3 className="font-semibold mt-2">
+              <label htmlFor="custom-calibration-date">
+                Custom calibration date
+              </label>
+            </h3>
+
+            <div className="w-full my-3 flex">
+              <DateInput
+                value={state.customCalibrationDate || ''}
+                onChange={customCalibrationDate =>
+                  setCalibrationDate(customCalibrationDate || undefined)
+                }
+              />
+            </div>
+
+            <p className="text-light-gray mb-2">
+              Setting a custom calibration date allows you to perform historical
+              simulations. The calibration date provides a rough starting point
```

Comment:
I suggest phrasing as "reference point" instead of "starting point", otherwise the next sentence suggests we will always start the simulation at the most recent date.

Reply:
Reworded for more clarity.